app/models/user.rb

```diff
@@ -14,7 +14,7 @@ def self.from_omni_auth(auth_hash)
   end
 
   def self.top_users
-    ActiveRecord::Base.connection.execute(
+    connection.execute(
```

Comment:
**Code Quality-Nitpick**
Since `ActiveRecord::Base` is inherited, no need to state it.